lib/bearer-auth-middleware.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const jwt = require('jsonwebtoken');
+const createError = require('http-errors');
+const debug = require('debug')('parkify:bearer-auth');
+
+const User = require('../model/user.js');
+
+module.exports = function(req, res, next) {
```

Comment:
Just for consistency, can you change `req` and `res` to `request` and `response`.